aspnet/signalr/overview/getting-started/tutorial-server-broadcast-with-signalr.md

```diff
@@ -1,407 +1,495 @@
-
-The application that you'll build in the first part of this tutorial displays a grid with stock data.
-
-![StockTicker initial version](tutorial-server-broadcast-with-signalr/_static/image1.png)
-
-Periodically the server randomly updates stock prices and pushes the updates to all connected clients. In the browser the numbers and symbols in the **Change** and **%** columns dynamically change in response to notifications from the server. If you open additional browsers to the same URL, they all show the same data and the same changes to the data simultaneously.
-
-This tutorial contains the following sections:
-
-- [Prerequisites](#prerequisites)
-- [Create the project](#createproject)
-- [Set up the server code](#server)
-- [Set up the client code](#client)
-- [Test the application](#test)
-- [Enable logging](#enablelogging)
-- [Install and review the full StockTicker sample](#fullsample)
-- [Next steps](#nextsteps)
-
-The [Microsoft.AspNet.SignalR.Sample](http://nuget.org/packages/microsoft.aspnet.signalr.sample) NuGet package installs a sample simulated stock ticker application in a Visual Studio project.
+This tutorial shows how to create a web application that uses ASP.NET SignalR 2 to provide server broadcast functionality. Server broadcast means that the server starts the communications sent to clients.
 
-> [!NOTE]
-> If you don't want to work through the steps of building the application, you can install the SignalR.Sample package in a new Empty ASP.NET Web Application project. If you install the NuGet package without performing the steps in this tutorial, **you must follow the instructions in the readme.txt file**. To run the package you need to add an OWIN startup class which calls the ConfigureSignalR method in the installed package. You will receive an error if you do not add the OWIN startup class.
+The application that you'll create in this tutorial simulates a stock ticker, a typical scenario for server broadcast functionality. Periodically, the server randomly updates stock prices and broadcast the updates to all connected clients. In the browser, the numbers and symbols in the **Change** and **%** columns dynamically change in response to notifications from the server. If you open additional browsers to the same URL, they all show the same data and the same changes to the data simultaneously.
 
+> [!IMPORTANT]
+> If you don't want to work through the steps of building the application, you can install the SignalR.Sample package in a new Empty ASP.NET Web Application project. If you install the NuGet package without performing the steps in this tutorial, you must follow the instructions in the *readme.txt* file. To run the package you need to add an OWIN startup class which calls the `ConfigureSignalR` method in the installed package. You will receive an error if you do not add the OWIN startup class.
```

Comment:
Do these instructions and the tutorial build-from-scratch instructions both work?

Reply:
Yes, I was able to get both of these to work with VS 2017.